docs(firestore-counter): android client example (#541)
firebase/extensions: firestore-counter/POSTINSTALL.md

}
```

#### Client samples for incrementing counter and retrieving its value

##### Web Client

1.  Download and copy the [compiled client sample](https://github.com/firebase/extensions/blob/master/firestore-counter/clients/web/dist/sharded-counter.js) into your application project.

> 2.  Use the client sample in your code to increment counters. The code snippet below shows an example of how to use it. To see the full reference implementation, refer to the sample's TypeScript [source code](https://github.com/firebase/extensions/blob/master/firestore-counter/clients/web/src/index.ts).

Reviewer: ```suggestion
1.  Use the client sample in your code to increment counters. The code snippet below shows an example of how to use it. To see the full reference implementation, refer to the sample's TypeScript [source code](https://github.com/firebase/extensions/blob/master/firestore-counter/clients/web/src/index.ts).
```

  </html>
  ```

##### Android Client

> 1. Follow the Firebase android setup [guide](https://firebase.google.com/docs/android/setup) in order to use Firebase in your app.

Reviewer 2: ```suggestion
1. Follow the steps in [Add Firebase to your Android project](https://firebase.google.com/docs/android/setup) to use Firebase in your app.
```

> 2. Copy and paste the sample [code](https://github.com/firebase/extensions/blob/next/firestore-counter/clients/android/src/main/java/com/firebase/firestore/counter/FirestoreShardedCounter.java) and create this file  `FirestoreShardedCounter.java` in the relevant directory you wish to use the `FirestoreShardedCounter` instance.

Reviewer 2: ```suggestion
2. Copy the [sample code](https://github.com/firebase/extensions/blob/next/firestore-counter/clients/android/src/main/java/com/firebase/firestore/counter/FirestoreShardedCounter.java) to the directory in which you want to use the `FirestoreShardedCounter` instance.
```
